python/cuspatial/cuspatial/core/spatial/distance.py

```diff
@@ -109,6 +109,48 @@ def haversine_distance(p1: GeoSeries, p2: GeoSeries):
     -------
     result : cudf.Series
         The distance between pairs of points between `p1` and `p2`
+    Examples
+    --------
+    You can calculate the Haversine Distance from a cuDF dataframe or from a GeoSeries list of points
```

Comment:
I think the below only mentions computing distance from a cuDF dataframe, not from a list of points. Can omit the last part.

```suggestion
    You can calculate the Haversine Distance from a cuDF dataframe.
```